//package core.io.protocol;
import "DseSingleSpiritInfo.proto";
import "EquipmentInfo.proto";

message DetailSingleSoldier
{
	optional DseSingleSpiritInfo spiritinfo = 1;//式神信息
	repeated EquipmentInfo equipmentlist	= 2;//装备信息
}

message DseQueryUserInfo
{
	optional int32	res				= 1;
	optional string uid				= 2;
	optional string name			= 3;//玩家名字
	optional int32 level			= 4;//玩家等级
	optional int32 vipLevel			= 5;//VIP等级
	optional int32 head				= 6;//玩家头像
	optional int32 power			= 7;//玩家战斗力
	optional string guildname		= 8;//公会阴阳寮名字
	optional int32	usedsuitid		= 9;//玩家当前使用的时装id
	repeated DetailSingleSoldier	spiritlist = 10;//上阵式神列表
}

